### ISSUE TITLE
Error: Failed to emit precompiled header...

### ISSUE THREAD

**Issue by Ankit8946** [leavez/cocoapods-binary#77]
- What's your environment:
 CocoaPods : 1.6.0
Xcode : 10.2.1 (10E1001)
cocoapods-binary      : 0.4.4
plugin 'cocoapods-binary'

<img width="1065" alt="Screenshot 2019-05-28 at 4 35 38 PM" src="https://user-images.githubusercontent.com/14289212/58535062-8427de80-820a-11e9-98bd-c44389fc3907.png">

**Comment by pikacode** [comment]
same issue

**Comment by namel3ssguy** [comment]
Did you guys fix this? @Ankit8946 @pikacode

**Comment by pikacode** [comment]
> Did you guys fix this? @Ankit8946 @pikacode

yes. u can try reinstall pods, clear project, or update cocoapods's version
